tests/test_multidict_benchmarks.py

```diff
@@ -410,3 +410,56 @@ def test_multidict_repr(benchmark: BenchmarkFixture) -> None:
     @benchmark
     def _run() -> None:
         repr(md)
+
+
+def test_create_empty_multidict(benchmark: BenchmarkFixture) -> None:
+    @benchmark
+    def _run() -> None:
+        MultiDict()
```

Comment:
Would something like this work?
```suggestion
def test_create_empty_multidict(
    any_multidict_class: Type[MutableMultiMapping[str]],
    benchmark: BenchmarkFixture,
) -> None:
    benchmark(lambda: any_multidict_class())
```

Reply:
Yeah I think so

Reply:
I'll do a followup in a bit after I finish digging though the latest profiles

Reply:
Thanks! I figured it'd be useful to expand the tests in this way due to built-in parametrization.

Reply:
https://github.com/aio-libs/multidict/pull/1092